noir-projects/noir-contracts/contracts/multisig_contract/src/main.nr

```diff
@@ -95,15 +97,27 @@ contract MultiSigAccount {
         ]
         )[0] as u8;
         assert(app_auths >= threshold, "App payload not authorized by multisig owners");
-        
+
         // TODO: Should we emit the authwit as a nullifier to prevent from replaying the same payload?
         app_payload.execute_calls(&mut context);
+        if (app_payload.function_calls[0].function_selector != update_config_selector) {
```

Comment:
```suggestion
        // We don't update the nonce if it's an update config since we do it in a single operation in that method
        if (app_payload.function_calls[0].function_selector != update_config_selector) {
```